stable/selenium/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: selenium
-version: 0.4.0
-appVersion: 3.12.0
+version: 0.4.1
+appVersion: 3.12.1
```

Comment:
Please update version in readme as well an bump minor chart version.

Reply:
No that probably doesn't make sense. You even went behind the previous version. It's ok to update the version, but you should also do that in the readme.